src/pages/RequestView/RequestView.tsx

```diff
@@ -83,7 +83,7 @@ const RequestView = () => {
   }
 
   return (
-    <Box>
+    <>
```

Comment:
Again with the point of top-level components being rooted, but this file appears to have not been touched. So it probably doesn't matter one way or the other since I doubt it will exist in its current form after refactor.